pkg/proc/native/proc_windows.go

```diff
@@ -141,15 +141,30 @@ func findExePath(pid int) (string, error) {
 	}
 }
 
+var debugPrivilegeAcquired = false
```

Comment:
nit: s/debugPrivilegeAcquired/debugPrivilegeRequested/

I know this is a nit but technically we're not certain to have acquired this permission, only certain that we've requested it.

Reply:
Done.